Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -56,7 +61,7 @@ export function initialize(dotnetAdapter, element, elementId, options) {
         },
         time_24hr: options.timeAs24hr ? options.timeAs24hr : false,
         clickOpens: !(options.readOnly || false),
-        disable: options.disabledDates || [],
+        disable: disableDatesOptions.concat(disableDaysOptions),
```

Comment:
have we tried with 2 functions, and it works correctly?

Reply:
I've tried it locally.